### ISSUE TITLE
Is it possible to reorder a column group?

### ISSUE THREAD

**Issue by bawaji94** [schrodinger/fixed-data-table-2#298]

**Comment by wcjordan** [comment]
It is possible, but we don't provide a drag handle for it, so you will need to add your own to your grouped header renderer and handle the reorder logic using the column props.

@quixotically may have some advice, she implemented the solution for our application.  But it required sync'ing multiple grids to support grouped and non-grouped columns at the same time.

![livedesign - edited](https://user-images.githubusercontent.com/1034455/36541726-cde34e70-17ac-11e8-8237-79a7a902c03a.gif)

**Comment by quixotically** [comment]
Hello @bawaji94 ,
Yes, we use a separate grid for each header in our application. Because of some existing limitations of the ColumnGroups feature of FDT, we rolled our own. The way ours works is that we override some of the FDT css so that the nongrouped top header cells can span into the bottom header when needed. We render blank cells underneath in the bottom header to keep the spacing synced with the top header. Also, we only enabled the reorder handle in the top header (which includes the tall cells), which helped. But it was still a pretty involved change to get a custom group header working.

**Comment by Ninjinka** [comment]
Is it possible to view how you handle it? (the example showed in the gif)

**Comment by fonpacific** [comment]
Yeah, it would be nice to see some code on how to reorder column groups.

**Comment by wcjordan** [comment]
Our new reordering API should make this easier to implement.  Adding v1.2 milestone so we consider this when tackling.
